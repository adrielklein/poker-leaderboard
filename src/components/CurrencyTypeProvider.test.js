import React from 'react';
import { CurrencyTypeProvider } from './CurrencyTypeProvider';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

it('renders component', () => {
  const component = shallow(<CurrencyTypeProvider for={ ['column1', 'column2'] }/>);
  expect(toJson(component)).toMatchSnapshot();
});
