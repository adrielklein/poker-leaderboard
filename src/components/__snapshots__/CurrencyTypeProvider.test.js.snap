// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<DataTypeProvider
  availableFilterOperations={
    Array [
      "equal",
      "notEqual",
      "greaterThan",
      "greaterThanOrEqual",
      "lessThan",
      "lessThanOrEqual",
    ]
  }
  editorComponent={[Function]}
  for={
    Array [
      "column1",
      "column2",
    ]
  }
  formatterComponent={[Function]}
/>
`;
